modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector.adoc

```diff
@@ -0,0 +1,115 @@
+
+For more information, see the xref:mq::index.adoc[Anypoint MQ documentation set] and additional resources listed in <<See Also>>.
+
+== Prerequisites
+
+To use this connector, you should be familiar with Anypoint Studio, Anypoint connectors, and Mule flows. The Anypoint MQ connector is free, but connecting to the service in Anypoint Platform requires a license.
+
+=== Compatibility
+
+[%header%autowidth.spread]
+|===
+|Software |Version
+|Mule Runtime |4.1.1 and later
+|Anypoint Studio |v7 and later
+|===
+
+IMPORTANT: This documentation applies to Mule 4 only. For Mule 3 compatible connector information, see xref:3.9@mule-runtime::anypoint-mq-connector.adoc[Mule 3 MQ Connector].
+
+To migrate from the Mule 4 connector version 2.x to the Mule 4 connector version 3.x, see the xref:anypoint-mq/3.x/anypoint-mq-new-major.adoc[Anypoint MQ 3.x Connector Migration Guide].
+
+== Install Anypoint MQ Connector
+
+In Anypoint Studio:
+
+. Create or open a Mule project.
+. Click the Exchange *(X)* icon in the Studio task bar.
+. Enter your Anypoint Platform username and password and click *Sign in*.
+. Select *All assets* and search for "MQ"
+. Select "Anypoint MQ Connector - Mule 4", and click *Add to project*.
+. Follow the prompts to install the connector.
+
+For standalone apps or apps written in XML:
+
+. Open your Mule project in Anypoint Studio.
+. Add the connector as a dependency in the pom.xml file:
++
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>anypoint-mq-connector</artifactId>
+  <version>${RELEASE}</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+For the latest POM, Gradle, or SBT dependencies, see the  https://www.anypoint.mulesoft.com/exchange/com.mulesoft.connectors/anypoint-mq-connector/[Exchange Anypoint MQ Connector] asset. Click *Dependency Snippets* for a complete list.
+
+
+== Configure Anypoint MQ in a Mule Project
+
+In Anypoint Studio:
+
+. Install the connector, if it's not already installed. See <<Install Anypoint MQ Connector>>.
+. Click *File > New > Mule Project* to create a new project in Studio.
+. Search for "mq" in the *Mule Palette* to see all the available operations for the *Anypoint MQ* connector.
+
+=== Create a New Configuration for the Anypoint MQ Connector
+
+. Drag the operation from the palette to the canvas, then select it.
+. Click the green plus sign to the right of *Connector configuration*.
+. Specify the broker URL for the region you want your MQ queues and exchanges to reside.
++
+To obtain and configure this URL, in Anypoint Platform, select *MQ > Destinations*. See xref:mq::mq-faq.adoc#regions[FAQ: Anypoint MQ] for the list of regions.
```

Comment:
Linking to the FAQ has no value + users aren't configuring a URL. The user can't configure in one region and run in another--what they need is to copy their current region to the UI. Plus when they click MQ, it automatically opens in the Destinations screen.
Better is to say "In **Anypoint Platform** > **MQ**, copy the URL for your region to the **URL** field."